book/conclusion/datacubes_revisited.md

```diff
@@ -83,7 +85,7 @@ A dataset must have a **dimensional coordinate** variable for each dimension in
 
 #### Relevant examples
 ##### 1. Handling time-varying metadata
-- Metadata that varies over `(time)` should be stored as coordinate variables along the `time` dimension.
+- Metadata that varies over `(time)` should be stored as coordinate variables along the `time` dimension (e.g. whether a scene was taken during an ascending or descending pass).
 - Metadata that varies over `time`, `x`, and `y` should be coordinate variables that exist along those dimensions.
```

Comment:
I didn't understand this line

Reply:
i agree out of context its weird, having the example is a lot better